bundle/src/main/java/com/adobe/acs/commons/remoteassets/impl/RemoteAssetsBinarySyncImpl.java

```diff
@@ -155,11 +155,6 @@ private void flagAssetAsFailedSync(ResourceResolver remoteAssetsResolver, Resour
             remoteAssetsResolver.commit();
         } catch (Exception e) {
             LOG.error("Error flagging remote asset '{}' as failed - asset may attempt to sync numerous times in succession", resource.getPath(), e);
-            try {
-                remoteAssetsResolver.revert();
```

Comment:
@joerghoh is this `.revert()` handled elsewhere now?

Reply:
this method is called from syncAsset only; and immediately after this call to flagAssetAsFailedSync() the resolver is closed.
No need to revert pending changes when the resolver is closed immediately after anyway.